services/wordpressService.ts

```diff
@@ -0,0 +1,114 @@
+import { Observer, ObserverCallback, ObserverDestructor } from "../../core/Observer";
+import { LogService } from "../../core/LogService";
+import { WordpressClient } from "../../core/wordpress/WordpressClient"
+import { isWordpressPageDTO, WordpressPageDTO } from "../../core/wordpress/dto/WordpressPageDTO"
+import { isWordpressPagesDTO, WordpressPageListDTO } from "../../core/wordpress/dto/WordpressPageListDTO"
+import { HgFrontend } from "../HgFrontend";
+import { WordpressReferenceDTO } from "../../core/wordpress/dto/WordpressReferenceDTO";
+import { WordpressUserProfileDTO } from "../../core/wordpress/dto/WordpressUserProfileDTO";
+
+HgFrontend.initialize(); // FIXME: Wasn't certain where this should be placed, but Im certain it needs to be fixed!
```

Comment:
Not here, no.  Never in the library code. 

It should got to the actual main program's `index.ts` or `ServiceInitializerService` if it has one, etc -- and called only once for each browser instance. 

E.g. it is never called from the `fi.hg.frontend` library. It initializes the browser code.

Reply:
Moved to index.ts